ding/bonus/config.py

```diff
@@ -4,92 +4,157 @@
+            cfg.n_sample = 1024
+        elif env == 'rocket_landing':
+            cfg.n_sample = 2048
+            cfg.adv_norm = False
+            cfg.model = dict(
+                encoder_hidden_size_list=[64, 64, 128],
+                actor_head_hidden_size=128,
+                critic_head_hidden_size=128,
+            )
+        elif env == 'drone_fly':
+            cfg.action_space = 'continuous'
+            cfg.adv_norm = False
+            cfg.epoch_per_collect = 5
+            cfg.learning_rate = 5e-5
+            cfg.n_sample = 640
+        elif env == 'hybrid_moving':
+            cfg.action_space = 'hybrid'
+            cfg.n_sample = 3200
+            cfg.entropy_weight = 0.03
+            cfg.batch_size = 320
+            cfg.adv_norm = False
+            cfg.model = dict(
+                encoder_hidden_size_list=[256, 128, 64, 64],
+                sigma_type='fixed',
+                fixed_sigma_value=0.3,
+                bound_type='tanh',
+            )
+        elif env == 'evogym_carrier':
+            cfg.action_space = 'continuous'
+            cfg.n_sample = 2048
+            cfg.batch_size = 256
+            cfg.epoch_per_collect = 10
+            cfg.learning_rate = 3e-3
+        elif env == 'mario':
+            cfg.n_sample = 256
+            cfg.batch_size = 64
+            cfg.epoch_per_collect = 2
+            cfg.learning_rate = 1e-3
+            cfg.model = dict(
+                encoder_hidden_size_list=[64, 64, 128],
+                critic_head_hidden_size=128,
+                actor_head_hidden_size=128,
+            )
+        elif env == 'di_sheep':
+            cfg.n_sample = 3200
+            cfg.batch_size = 320
+            cfg.epoch_per_collect = 10
+            cfg.learning_rate = 3e-4
+            cfg.adv_norm = False
+            cfg.entropy_weight = 0.001
+        elif env == 'procgen_bigfish':
+            cfg.n_sample = 16384
+            cfg.batch_size = 16384
+            cfg.epoch_per_collect = 10
+            cfg.learning_rate = 5e-4
+            cfg.model = dict(
+                encoder_hidden_size_list=[64, 128, 256],
+                critic_head_hidden_size=256,
+                actor_head_hidden_size=256,
+            )
+        elif env in ['atari_qbert', 'atari_kangaroo', 'atari_bowling']:
+            cfg.n_sample = 1024
+            cfg.batch_size = 128
+            cfg.epoch_per_collect = 10
+            cfg.learning_rate = 0.0001
+            cfg.model = dict(
+                encoder_hidden_size_list=[32, 64, 64, 128],
+                actor_head_hidden_size=128,
+                critic_head_hidden_size=128,
+                critic_head_layer_num=2,
+            )
+        else:
+            raise KeyError("not supported env type: {}".format(env))
+    elif algorithm == 'TD3':
+        cfg = TD3Policy.default_config()
+        if env == 'hopper':
+            cfg.update(
+                dict(
+                    exp_name='hopper_td3',
```

Comment:
simplify config, remove all the unmodifed fields

Reply:
fixed